server/views/partials/layout.njk

```diff
@@ -32,7 +32,201 @@
+      !function (v, y, T) {
```

Comment:
Let's either put this in a formatting commit, or undo this change and have it all compressed (personal preference for the latter to reduce the amount of scrolling as this is a script copied from Azure and we probably won't need to debug/tweak it (famous last words...)

Reply:
The issue above is resolved, but could you take this formatting out of this commit please?